interactive-bduf-orchestrator/src/shared/errors/async-utils.ts

```
@@ -0,0 +1,354 @@
+
+        // Calculate delay with exponential backoff
+        let delay = Math.min(baseDelay * Math.pow(factor, attempt), maxDelay);
+        
+        if (jitter) {
+          delay *= (0.5 + Math.random() * 0.5);
+        }
+
+        await new Promise(resolve => setTimeout(resolve, delay));
+      }
+    }
+  });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)